src/picongpu/include/particles/Particles.tpp

```diff
@@ -72,7 +73,9 @@ datasetID( datasetID )
 
     log<picLog::MEMORY > ( "size for all exchange = %1% MiB" ) % ( (double) sizeOfExchanges / 1024. / 1024. );
 
-    const uint32_t commTag = FrameType::CommunicationTag + SPECIES_FIRSTTAG;
+    const uint32_t commTag = PMacc::traits::GetUniqueTypeId<FrameType>::uid + SPECIES_FIRSTTAG;
```

Comment:
I will add a check (in a new pull request) in our communicator. IMO that is the only place were it can be handled.